.golangci.yml

```diff
@@ -1,4 +1,4 @@
-# v1.55.2
+# v1.56.2
```

Comment:
I think this shouldn't be in this PR ;)

Reply:
Yeah, agree. But the linter is failing :(

Reply:
Removed the bumping commit.

Reply:
Now it works, weird.